### ISSUE TITLE
Killed error when using  nsg-kitti-car-depth-recon pre-trained model.

### ISSUE THREAD

**Issue by gallenszl** [OPEN-AIR-SUN/mars#59]
hello, when i run the render process with the pretrained nsg-kitti-car-depth-recon model, i.e., 
`python scripts/cicai_render.py --load-config /data/download/code/mars/outputs/nsg-kitti-car-depth-recon/2023-05-30_182941/config.yml --output-path renders/result_1.mp4`
I met the following error:
```
[[20:43:51] [array([0.]), array([1.]), array([2.]), array([7.]), array([8.]), array([3.]),          nsg_dataparser.py:843
           array([4.]), array([9.]), array([5.]), array([12.]), array([6.]), array([10.]),                              
           array([11.]), array([13.]), array([14.])] in this scene.                                                     
finished data parsing
[20:43:59] [array([0.]), array([1.]), array([2.]), array([7.]), array([8.]), array([3.]),          nsg_dataparser.py:843
           array([4.]), array([9.]), array([5.]), array([12.]), array([6.]), array([10.]),                              
           array([11.]), array([13.]), array([14.])] in this scene.                                                     
Killed]
```
However the render process can be successfully finished with the kitti_nvs_75 model, i.e., 
`python scripts/cicai_render.py --load-config /data/download/code/mars/outputs/kitti_nvs_75/nsg-kitti-car-depth-nvs/2023-07-27_021831/config.yml --output-path renders/result.mp4`

Such a situation seems a little wired and could you help me locate the problem? My gpu is nvidia 3090 with 24G memory.

**Comment by xBeho1der** [comment]
Hi! Thanks for your reply. Maybe you should try shortening your training sequence. We load the training dataset on CPU instead of GPU.

**Comment by Gniniug123** [comment]
Hi, hello, I also encountered the same problem as you. Where can I modify the training sequence? Is it in the config.yaml I loaded? Which parameter is it corresponding to?

**Comment by xBeho1der** [comment]
@Gniniug123 

If you met such an issue while training, you can modify `first_frame` and `last_frame` in `cicai_config.py`. If it occurred while rendering, you can modify the `config.yaml` you loaded.

**Comment by Gniniug123** [comment]
Thank you very much for your answer. Reducing the values of these two parameters can indeed avoid this problem, but I would like to know more about the specific meaning of these two parameters? What is the reasonable value for them to set?

**Comment by Gniniug123** [comment]
@xBeho1der Sorry, I have to bother you. When I wanted to shorten the image sequence during training, I didn't find a place in cicai_config.py where I could modify first_frame last_frame：
KITTI_Recon_Mars_Car_Depth = MethodSpecification(
    config=TrainerConfig(
        method_name="mars-kitti-car-depth-recon",
        steps_per_eval_image=STEPS_PER_EVAL_IMAGE,
        steps_per_eval_all_images=STEPS_PER_EVAL_ALL_IMAGES,
        steps_per_save=STEPS_PER_SAVE,
        max_num_iterations=MAX_NUM_ITERATIONS,
        save_only_latest_checkpoint=False,
        mixed_precision=False,
        use_grad_scaler=True,
        log_gradients=True,
        pipeline=MarsPipelineConfig(
            datamanager=MarsDataManagerConfig(
                dataparser=MarsKittiDataParserConfig(
                    use_car_latents=True,
                    use_depth=True,
                    car_object_latents_path=Path(
                        "/DATA_EDS/liuty/ckpts/pretrain/car_nerf/latent_codes_car_van_truck.pt"
                    ),
                    split_setting="reconstruction",
                    car_nerf_state_dict_path=Path("/DATA_EDS/liuty/ckpts/pretrain/car_nerf/epoch_670.ckpt"),
                    scale_factor=0.1,
                ),
                train_num_rays_per_batch=4096,
                eval_num_rays_per_batch=4096,
                camera_optimizer=CameraOptimizerConfig(mode="off"),
            ),
            model=SceneGraphModelConfig(
                background_model=NerfactoModelConfig(),
                object_model_template=CarNeRFModelConfig(_target=CarNeRF),
                object_representation="class-wise",
                object_ray_sample_strategy="remove-bg",
                mono_depth_loss_mult=0.01,
                depth_loss_mult=0,
            ),
        ),
        optimizers={
            "background_model": {
                "optimizer": RAdamOptimizerConfig(lr=1e-3, eps=1e-15),
                "scheduler": ExponentialDecaySchedulerConfig(lr_final=1e-5, max_steps=200000),
            },
            "learnable_global": {
                "optimizer": RAdamOptimizerConfig(lr=1e-3, eps=1e-15),
                "scheduler": ExponentialDecaySchedulerConfig(lr_final=1e-5, max_steps=200000),
            },
            "object_model": {
                "optimizer": RAdamOptimizerConfig(lr=5e-3, eps=1e-15),
                "scheduler": ExponentialDecaySchedulerConfig(lr_final=1e-5, max_steps=200000),
            },
        },
        # viewer=ViewerConfig(num_rays_per_chunk=1 << 15),
        vis="wandb",
    ),
    description="Neural Scene Graph implementation with vanilla-NeRF model for backgruond and object models.",
)

**Comment by xBeho1der** [comment]
Hi! @Gniniug123 

Just add it in `MarsKittiDataParserConfig`

![image](https://github.com/OPEN-AIR-SUN/mars/assets/107318439/652c99fd-b4d0-4246-a87b-081c7bfcd5ac)

**Comment by Gniniug123** [comment]
@xBeho1der Thank you very much for your answer. I can now train normally.
